templates/definition/meter/fronius-solarapi-v1.yaml

```diff
@@ -48,6 +48,11 @@ render: |
   {{- end }}
   {{- if eq .usage "battery" }}
     jq: if .Body.Data.Site.P_Akku == null then 0 else .Body.Data.Site.P_Akku end
+  energy:
+    source: http
+    uri: http://{{ .host }}/solar_api/v1/GetPowerFlowRealtimeData.fcgi
+    jq: if .Body.Data.Site.E_Total == null then 0 else .Body.Data.Site.E_Total end
+  {{- if .password }}
```

Comment:
?

Reply:
![image](https://github.com/user-attachments/assets/13f42918-fc9d-418a-a123-d65d9c2fe7ce)
should add the "Energy", as it is since https://github.com/evcc-io/evcc/pull/19095 available at fronius gen24 inverters, as well for solar api requested inverters

Reply:
I repeat: what has password to do with it? Check Tests- this template is currently broken.